database/migrations/2021_01_24_170653_create_reservations_table.php

```diff
@@ -44,6 +44,7 @@ public function down()
     {
         Schema::dropIfExists('reservations');
 
+        \App\Models\BookingRequest::truncate();
```

Comment:
errr, this smells like a very bad practice. Why is truncation needed?

Reply:
because while we undo reservations we can't convert them back to bookings since restrictions allow bookings to have multiple times. When rolling back, it can only have one time at a time. Also we are dropping the reservations table already. because of this we need to truncate the bookings requests so that the foreign keys aren't null. All live booking requests that exist should be deleted when you delete all the times associated with them

Reply:
Explanation checks out 👍